package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "shaka-player",
+  "name": "stream-shaka-player",
   "description": "DASH/EME video player library",
-  "version": "4.3.0",
-  "homepage": "https://github.com/shaka-project/shaka-player",
+  "version": "4.3.0-testout",
```

Comment:
always update this version before new release otherwise NPM publish step will err out saying same version was already release